Mlem/Models/Content/Community/Community List/CommunityListModel.swift

```diff
@@ -15,38 +14,45 @@ class CommunityListModel: ObservableObject {
     @Dependency(\.errorHandler) var errorHandler
     @Dependency(\.favoriteCommunitiesTracker) var favoriteCommunitiesTracker
     @Dependency(\.notifier) var notifier
-    @Dependency(\.mainQueue) var mainQueue
-    
-    @Published private(set) var communities = [APICommunity]()
     
-    private var subscriptions = [APICommunity]()
-    private var favoriteCommunities = [APICommunity]()
     private var cancellables = Set<AnyCancellable>()
     
+    @Published private(set) var allSections: [CommunityListSection] = .init()
+    @Published private(set) var visibleSections: [CommunityListSection] = .init()
+    
+    private(set) var subscribed: [APICommunity] = .init()
+    private var subscribedSet: Set<Int> = .init()
+    private(set) var favorited: [APICommunity] = .init()
+    
     init() {
         favoriteCommunitiesTracker
             .$favoritesForCurrentAccount
             .dropFirst()
```

Comment:
What's `dropFirst` for?

Reply:
🤔 good question. Once upon a time it might have served a purpose, but I can't find one any more.